packages/@webex/internal-plugin-metrics/src/new-metrics.ts

```diff
@@ -63,6 +67,10 @@ class Metrics extends WebexPlugin {
       this.callDiagnosticMetrics = new CallDiagnosticMetrics({}, {parent: this.webex});
       // @ts-ignore
       this.behavioralMetrics = new BehavioralMetrics({}, {parent: this.webex});
+      // @ts-ignore
```

Comment:
IMO - the actual `creation` of these objects *should be* configurable, i.e., we dont NEED to create a businessMetrics object if we are never going to use it. The consumer should be able to decide if these are needed.

Reply:
So you're thinking about extra config flags when we instantiate the plugin/SDK?

Reply:
Or more a lazy-apporach: we create the metric instance the first time the user try to send an event -- which makes more sense I guess.

Reply:
lazy approach works fine for me as well, however, there's 2 usecases

1. pre-warming (use config to get the object already created, but just the ones the consumer wants)
2. lazy created similar to a JIT-C, which is great

However, the performance impact of something like this is so infinitesmal it's hard to argue about it

tldr: i'm cool with a lazy approach